ipaserver/install/server/install.py

```diff
@@ -1130,6 +1130,12 @@ def uninstall(installer):
                           ' # getcert stop-tracking -i <request_id>\n'
                           'for each id in: %s' % ', '.join(ids))
 
+    # Remove the cert renewal lock file
+    try:
+        os.remove(paths.IPA_RENEWAL_LOCK)
+    except Exception:
+        pass
```

Comment:
This would be better (off the top of my head):
```python
    try:
        os.remove(paths.IPA_RENEWAL_LOCK)
    except OSError as e:
        if e.errno != errno.ENOENT:
            root_logger.warning("Failed to remove %s: %s", paths.IPA_RENEWAL_LOCK, e)
```